book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def add_rental(date)
+    rentals.push(date, self)
+  end
```

Comment:
- [ ] KIndly make sure that  the Getters and Setters for @rentals is be present 👇🏻 

  `attr_accessor :title, :author, :rentals`

Reply:
I will add the correct getter and setter

Reply:
Well resolved @d4nQw3rty 👍🏽